src/components/SelectList/SelectList.tsx

```diff
@@ -131,21 +130,41 @@ const customStyles: StylesConfig = {
             ...disabled
         };
     },
-    option: (provided, state) => ({
-        ...provided,
-        color: state.isSelected ? Colors.WHITE : Colors.AUTHENTIC_BLUE_900,
-        padding: '0.375rem 0.75rem',
-        fontSize: 'inherit',
-        backgroundColor: state.isSelected
-            ? Colors.ACTION_BLUE_900
-            : state.isFocused
-            ? Colors.AUTHENTIC_BLUE_50
-            : Colors.WHITE,
-        '&:active': {
-            background: state.isSelected ? Colors.ACTION_BLUE_900 : Colors.AUTHENTIC_BLUE_50
-        },
-        wordWrap: 'break-word'
-    }),
+    option: (provided, state) => {
+        const colorsByState = {
+            isDisabled: {
+                color: Colors.AUTHENTIC_BLUE_350
+            },
+            isFocused: {
+                backgroundColor: Colors.ACTION_BLUE_50
+            },
+            isSelected: {
+                backgroundColor: Colors.ACTION_BLUE_900,
+                color: Colors.WHITE
+            }
+        };
+
+        const defaultColors = {
+            color: Colors.AUTHENTIC_BLUE_900,
+            backgroundColor: Colors.WHITE
+        };
+
+        const colors = Object.keys(colorsByState)
+            .filter(key => state[key])
+            .reduce((acc, style) => ({ ...acc, ...colorsByState[style] }), defaultColors);
+
+        return {
+            ...provided,
+            ...colors,
+            '&:active': {
+                ...colors
+            },
+            padding: `${state.selectProps.size === 'medium' ? '0.5rem' : '0.375rem'} 0.75rem`,
```

Comment:
While I understand that spaces should be only used on margins I wonder if paddings should have a certain guideline, padding of `0.375rem` looks arbitrary. @agonzalezUX any thoughts on this?

Reply:
we don't have paddings set in our library but they should follow the same size and structure as the margins.

Reply:
here we should use or spacing (padding) 1 (0.5rem) or half of it which would be 0.25rem

Reply:
@agonzalezUX that means we need to make sure [these types of designs](https://app.zeplin.io/project/57fe438f81b870887cc71736/screen/605c7840aebc4544301abe77) get updated.

For small will be `0.25rem`, medium `0.5rem` and large `1rem`

Reply:
ohh wait, my bad. Please forget what I said. I think that before use the same scale as the spacing we need to review that this is matching with the components that has padding so, for now I would not change it and would keep using what is in the zeplin file. Sorry, my fault 🙏🏼